sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_web_pub_sub_service_client.py

```diff
@@ -0,0 +1,209 @@
+    :ivar web_pub_sub: WebPubSubOperations operations
+    :vartype web_pub_sub: azure.messaging.webpubsubservice.operations.WebPubSubOperations
+    :keyword connection_string: connection string needed for the client to connect to Azure.
+    :paramtype connection_string： str
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: Union[~azure.core.credentials.TokenCredential, ~azure.core.credentials.AzureKeyCredential]
+    :keyword endpoint: Service URL. Default value is ''.
+    :paramtype endpoint: str
+    """
+
+    def __init__(
+        self,
+        credential,  # type: Union[TokenCredential, AzureKeyCredential]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        endpoint = kwargs.pop('endpoint', "")  # type: str
+        self._config = WebPubSubServiceClientConfiguration(credential, **kwargs)
+        self._client = PipelineClient(base_url=endpoint, config=self._config, **kwargs)
+
+        self._serialize = Serializer()
+        self._deserialize = Deserializer()
+        self._serialize.client_side_validation = False
+        self.health_api = HealthApiOperations(self._client, self._config, self._serialize, self._deserialize)
+        self.web_pub_sub = WebPubSubOperations(self._client, self._config, self._serialize, self._deserialize)
+
+
+    def send_request(
+        self,
+        request,  # type: HttpRequest
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> HttpResponse
+        """Runs the network request through the client's chained policies.
+
+        >>> from azure.core.rest import HttpRequest
+        >>> request = HttpRequest("GET", "https://www.example.org/")
+        <HttpRequest [GET], url: 'https://www.example.org/'>
+        >>> response = client.send_request(request)
+        <HttpResponse: 200 OK>
+
+        For more information on this code flow, see https://aka.ms/azsdk/python/protocol/quickstart
+
+        :param request: The network request you want to make. Required.
+        :type request: ~azure.core.rest.HttpRequest
+        :keyword bool stream: Whether the response payload will be streamed. Defaults to False.
+        :return: The response of your network call. Does not do error handling on your response.
+        :rtype: ~azure.core.rest.HttpResponse
+        """
+
+        request_copy = deepcopy(request)
+        request_copy.url = self._client.format_url(request_copy.url)
+        return self._client.send_request(request_copy, **kwargs)
+
+    def close(self):
+        # type: () -> None
+        self._client.close()
+
+    def __enter__(self):
+        # type: () -> WebPubSubServiceClient
+        self._client.__enter__()
+        return self
+
+    def __exit__(self, *exc_details):
+        # type: (Any) -> None
+        self._client.__exit__(*exc_details)
+
+    @staticmethod
+    def build_authentication_token(endpoint, hub, **kwargs):
```

Comment:
I don't think `build_authentication_token` should be a staticmethod on the client, it used to be it's own method. @johanste is this correct? Also do we still want this builder given that this SDK is no longer purely request builders and send_request?

Reply:
No, it should not be a `@staticmethod` on the client. Static methods are rarely to be used. Module level methods are available in python.

Yes, we still want the method.